src/NuGet.Services.EndToEnd/Support/Package.cs

```diff
@@ -68,6 +68,23 @@ public static Package Create(PackageCreationContext context)
                                context.Properties);
         }
 
+        public static Package Create(string path, PackageProperties packageProperties)
+        {
```

Comment:
Null check for the arguments?

Reply:
I can add them if you feel strongly. This is test code so I don't find it as important...